src/js/frontend/src/components/ChartTitle.js

```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import ReactDOM from "react-dom"
+import Radium from "radium"
+import { Link } from 'react-router'
+
+function ChartTitle (props) {
+  let description = this.props.sourceName ?
```

Comment:
make sure you change `this.props` to just `props` as it's not passed as a function param